cmd/cape/cmd/token.go

```diff
@@ -215,9 +215,64 @@ This is different than creating a token that gives access to an individual funct
 	},
 }
 
+var deleteTokenCmd = &cobra.Command{
+	Use:   "delete",
+	Short: "Delete tokens from your account",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		verbose, err := cmd.Flags().GetBool("verbose")
+		if err != nil {
+			return err
+		}
+
+		tokenIDStr := args[0]
+		url := C.EnclaveHost
+
+		req, err := http.NewRequest(http.MethodDelete, fmt.Sprintf("%s/v1/token/%s", url, tokenIDStr), nil)
+		if err != nil {
+			return err
+		}
+
+		authToken, err := authToken()
+		if err != nil {
+			return err
+		}
+
+		var bearer = "Bearer " + authToken
```

Comment:
nit: why is this a variable? Be a bit more cleaner to just embed it in the below line.